src/System.Management.Automation/FormatAndOutput/common/FormatViewGenerator.cs

```diff
@@ -348,19 +348,70 @@ protected class DataBaseInfo
 
         protected DataBaseInfo dataBaseInfo = new DataBaseInfo();
 
-        protected List<MshResolvedExpressionParameterAssociation> activeAssociationList = null;
+        private List<MshResolvedExpressionParameterAssociation> _activeAssociationList;
+
+        /// <summary>
+        /// Gets the current active association list for read-only access.
+        /// </summary>
+        protected IReadOnlyList<MshResolvedExpressionParameterAssociation> ActiveAssociationList => _activeAssociationList;
```

Comment:
I'd prefer use exactly List type. We have no need to use generics and pay for virtual method.